CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ For details about compatibility between different releases, see the **Commitment
 
 ### Fixed
 
+- Listing entities for organizations (when using the experimental `is.bunstore` feature flag).
```

Comment:
I would like to cherry-pick this into the release candidate 2, so you may skip the changelog entry.